### PR TITLE
K8s integration

### PR DESCRIPTION
**Pull request by lmanohara**
-Initially development, assuming application is deployed in a first container.

### CODE REVIEW

File: modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/Container.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
     private String probe;
     private List<ServiceProxy> serviceProxies;
     private List<String> labels;
+    private List volumeMounts;
```
Comment (gnudeep) [inline modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/Container.java]: set the type.

File: modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java [hunk 2]
```diff
@@ -42,6 +42,6 @@
     public static final String RULES = "rules";
     public static final String HOST = "host";
 
-    public static final String VOLUME_MOUNT = "";
-    public static final String VOLUME_MOUNT_PATH = "";
+    public static final String VOLUME_MOUNT = "default";
+    public static final String VOLUME_MOUNT_PATH = "/etc/default";
```
Comment (gnudeep) [inline modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java]: Try to add generic paths.

Comment (lmanohara) [inline modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java]: This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/c707570879201754759203ff99b9c5ea5489dcba

File: modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/Container.java (hunk 1 above)
Comment (lmanohara) [inline modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/Container.java]: This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/6d720134d55b53819ec24559da1fe0cc7b9ce447

**Pull request merged**